onetimes.yml

```diff
@@ -31,4 +31,20 @@ events:
       minutes: 30
     description: |
       Sebastian Schuberth, founder of OSS Review Toolkit (ORT), will join us during our regular Team OPS meeting to present ORT and be available to answer your questions. You are welcome to send us your questions in advance so that we can prepare this meeting in the best possible way.
-    location: "https://conf.scs.koeln:8443/SCS-Tech"
\ No newline at end of file
+    location: "https://conf.scs.koeln:8443/SCS-Tech"
+  - summary: SIG standard/cert: IaaS deep dive
+    begin: 2022-08-04 14:05:00
+    duration:
+      minutes: 50
+    description: |
```

Comment:
I recommend to either remove the newlines or use a folded block scalar style (`>`) here in order to allow for dynamic word wrapping by the calendar client.